package.json

```diff
@@ -21,20 +21,18 @@
 	},
 	"scripts": {
 		"test": "npm-run-all --parallel lint test:unit",
-		"test:unit": "mocha test/test_*.js",
+		"test:unit": "node --test test/test_*.js",
 		"lint": "eslint --cache lib bin/* test test/bin/* && echo ✓"
 	},
 	"engines": {
 		"node": ">= 18"
 	},
 	"dependencies": {
 		"browserslist": "~4.22.1",
-		"minimist": "~1.2.7",
 		"nite-owl": "~5.0.5"
 	},
 	"devDependencies": {
 		"eslint-config-fnd": "^1.13.0",
-		"mocha": "^10.2.0",
```

Comment:
This is delightful! If I'm not mistaken, this should get rid of a whopping 130 dependencies - except _json5_ and _tsconfig-paths_ depend on minimist as well for some reason, so it's "just" 50 fewer dependencies.

On a related note, because both those packages appear to be transitive dependencies of _eslint-config-fnd_: I've [recently](https://github.com/FND/metacolon/commit/873a0b492d20fdbad476e97249b57561d702b0e0) decided not to bother upgrading the latter (which triggers deprecation warnings) because that's a major pita, plus that elaborate DIY style was never a good idea in the first place... 😬 So, uhm, apologies for sinking our ecosystem in favor of pixel perfection?

Reply:
Correct. It's [still a risk, but less so](https://prepitaph.org/articles/banishing-npm/#ref:replaceability). (Of course diff pollution seems likely either way.)

Reply:
Tracked as https://github.com/faucet-pipeline/faucet-pipeline-core/issues/145